draft-ietf-scitt-software-use-cases.md

```diff
@@ -127,11 +127,11 @@ Typically, the range of types of statements about digital products (and their de
 Taking the type and structure of all statements about digital and products into account might not be possible.
 Examples of statements may include commit signatures, build environment and parameters, software bill of materials, static and dynamic application security testing results, fuzz testing results, release approvals, deployment records, vulnerability scan results, and patch logs.
 In consequence, instead of trying to understand and describe the detailed syntax and semantics of every type of statement about digital products, the SCITT architecture focuses on ensuring statement authenticity, visibility/transparency, and intends to provide scalable accessibility.
-The following use case illustrates the scope of SCITT and elaborate on the generic problem statement above.
+The following use case illustrates the scope of SCITT and elaborates on the generic problem statement above.
 
 # Software Supply Chain Use Cases
 
-## Verification that Signing Certificate is Authorized by Supplier
+## Verification That Signing Certificate Is Authorized by Supplier
```

Comment:
```suggestion
## Verification that Signing Certificate is Authorized by Supplier
```

I think we go by a "everything is lowercase except the first word and proper nouns or verbs".

Reply:
I used: https://capitalizemytitle.com/